lib/IRGen/IRGen.cpp

```diff
@@ -1111,7 +1111,7 @@ struct LLVMCodeGenThreads {
 /// All this is done in multiple threads.
 static void performParallelIRGeneration(
     const IRGenOptions &Opts, swift::ModuleDecl *M, std::unique_ptr<SILModule> SILMod,
-    StringRef ModuleName, int numThreads,
+    StringRef ModuleName, unsigned int numThreads,
```

Comment:
Come to think of it, let's drop this parameter. You can get it from the options in the SIL module.